feature/models/label.js

```diff
@@ -0,0 +1,22 @@
+const mongoose = require('mongoose');
+const Schema = mongoose;
+
+const labelSchema = Schema({
+  name: {
+    type: String,
+    required: true,
+  },
+  description: {
+    type: String,
+    required: true,
+  },
+  color: {
+    type: String,
+    required: true,
+  },
+  issues: [],
+});
+
+const Label = mongoose.model('Label', labelSchema);
+
+module.exports = Label;
```

Comment:
If a PR is Targetting More than 1 Issue than it should be mentioned in the PR description

Reply:
Description added.